src/com/codefortomorrow/beginner/chapter4/examples/Arithmetic.java

```diff
@@ -4,21 +4,27 @@
 
     public static void main(String[] args) {
         // addition
-        int a = 5 + 1; // a has the value 6
+        int a = 5 + 1;
```

Comment:
Should we still keep the comment?
```suggestion
        int a = 5 + 1; //a has value 6
```

Reply:
No, I deleted the comments because if you run the program the results are printed anyway (I added print statements).